ethers-contract/tests/it/main.rs

```diff
@@ -9,8 +9,8 @@ mod contract_call;
 
 mod eip712;
 
-#[cfg(all(not(target_arch = "wasm32"), not(feature = "celo")))]
+// #[cfg(all(not(target_arch = "wasm32"), not(feature = "celo")))]
 mod common;
 
-#[cfg(all(not(target_arch = "wasm32"), not(feature = "celo")))]
+// #[cfg(all(not(target_arch = "wasm32"), not(feature = "celo")))]
```

Comment:
no longer needed?

Reply:
nope just forgot. i do it to enable rust analyzer because i have it on all-features, is there another way lol